src/apps/intel_mp/intel_mp.lua

```diff
@@ -31,6 +31,10 @@ local S           = require("syscall")
 local CallbackAlarm = alarms.CallbackAlarm
 local transmit, receive, empty = link.transmit, link.receive, link.empty
 
+-- This environment variable is for use in tests, where a random hash
+-- key is often undesirable (e.g., to ensure all queues see traffic in a test)
+local rss_hash_key = tonumber(lib.getenv("SNABB_RSS_HASH_KEY"))
```

Comment:
We already have an environment variable for making deterministic random numbers for tests: SNABB_RANDOM_SEED.